ansible/roles/java/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+# Java settings
+---
+java_home: "/opt/epics-tools/lib/jvm/jdk-17"
+mvn_home: "/opt/epics-tools/lib/apache-maven"
```

Comment:
Maybe you should have a separate maven role, its not needed for the archiver at least.

Reply:
Originally all was part of the Phoebus role.
I separated what I thought was not Phoebus-specific.

Will move Maven back to Phoebus for the time being.

Reply:
Done in c717885.